bin/test

```diff
@@ -8,11 +8,28 @@ require 'yaml'
 @verbose = false
 
 OptionParser.new do |opts|
-  opts.banner = "Usage: bin/test [options] <gem...>"
+  opts.banner = <<~BANNER
+    Usage: bin/test [options] <gem...>
+
+    Examples:
+        # Test a gem
+        $ bin/test gems/rack/2.2
+
+        # Test a gem with verbose messages
+        $ bin/test gems/rack/2.2 -v
+
+        # Test all the gems
+        $ bin/test -a
+
+    Options:
+  BANNER
 
   opts.on("-a", "--all", "Test all gems") do @all = true end
   opts.on("-v", "--verbose", "Print more messages") do @verbose = true end
```

Comment:
Isn't the default `--help` message enough? `bin/test --help` displays the help message by default even if it is not defined explicitly.

```ruby
$ bin/test --help
Usage: bin/test [options] <gem...>

Examples:
    # Test a gem
    $ bin/test gems/rack/2.2

    # Test a gem with verbose messages
    $ bin/test gems/rack/2.2 -v

    # Test all the gems
    $ bin/test -a

Options:
    -a, --all                        Test all gems
    -v, --verbose                    Print more messages
```

In this case, `--help` is not described in the `Options:` section. But I think it is not a large problem because the user knows `--help` option if the user see this message.

Reply:
Got it. I wonder if the help message is output regardless of `opts.on("-h", "--help")`. 😅